src/intro.js

```diff
@@ -1,3 +1,2 @@
 (function( jQuery, window, undefined ) {
```

Comment:
BTW, we could drop the `undefined`; Core doesn't protect itself from `undefined` shadowing anymore so there's no reason Migrate should.